src/nspawn/nspawn-patch-uid.c

```diff
@@ -300,6 +300,10 @@ static int is_procfs_sysfs_or_suchlike(int fd) {
                F_TYPE_EQUAL(sfs.f_type, PSTOREFS_MAGIC) ||
                F_TYPE_EQUAL(sfs.f_type, SELINUX_MAGIC) ||
                F_TYPE_EQUAL(sfs.f_type, SMACK_MAGIC) ||
+               F_TYPE_EQUAL(sfs.f_type, SECURITYFS_MAGIC) ||
+               F_TYPE_EQUAL(sfs.f_type, RAMFS_MAGIC) ||
```

Comment:
hmm, i think we should permit ramfs, for the same reasons as we permit tmpfs, no: it may container user data, and if the admin pre-mounts that, then we should patch it...